src/inflators/model.tsx

```diff
@@ -139,6 +141,20 @@ export function inflateModel(world: HubsWorld, rootEid: number, { model }: Model
     }
   });
 
+  // TODO: Not sure if here is the best place and
```

Comment:
This seems like a reasonable spot to set up the MixerAnimatable component and set up a default LoopAnimation component if one does not exist already.